Docker/ods-api-db-admin/alpine/mssql/README.md

```diff
@@ -1,7 +1,10 @@
 # Ed-Fi Admin Databases
 
-> **Note**
-> Microsoft SQL Server images are not provided by the Ed-Fi Alliance at this
-> time due to licensing restrictions. However, you can connect an external MS
+> [!NOTE]
+> An Ubuntu-based [Microsoft SQL Server image](../../ubuntu/mssql/README.md) is provided by the Ed-Fi Alliance
+> under the [SQL Express license](https://www.microsoft.com/en-us/Useterms/Retail/SQLServer2022/SQLServer2022DeveloperExpressEvaluation/Useterms_Retail_SQLServer2022_SQLServer2022DeveloperExpressEvaluation_English.htm). 
+> 
+> Also, you can connect an external MS
 > SQL Server to ODS / API and Admin App running on containers. Refer to [Docker
 > Deployment](https://techdocs.ed-fi.org/x/EIeIBg) for details.
+
```

Comment:
I left the folders thinking of users that may be looking for an Alpine implementation; now I believe that instead of having an Alpine folder with just a readme indicating there is no Alpine version, we can just mention that in the root Docker folder readme file